src/main/java/org/ibp/api/rest/germplasm/GermplasmNameResource.java

```diff
@@ -33,6 +38,24 @@
 		return new ResponseEntity<>(this.germplasmNameService.createName(programUUID, germplasmNameRequestDto, gid), HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "Create code name (CODE1, CODE2, CODE2) for specified list of germplasm")
+	@PreAuthorize("hasAnyAuthority('ADMIN', 'GERMPLASM', 'MANAGE_GERMPLASM', 'CODE_GERMPLASM')")
+	@RequestMapping(value = "/crops/{cropName}/germplasm/coding", method = RequestMethod.POST)
```

Comment:
I think we should use codes instead of coding to avoid the verb

Reply:
Done